### DISCUSSION TITLE
Application has not been successfully authorised

### DISCUSSION THREAD

**Discussion by 0byt3** [abraunegg/onedrive#1884] [Q&A] [answered]
I had an issue adding my work onedrive to my computer. It was receiving an "Application has not been successfully authorised. Please check your URI response entry and try again." error, so I tried updating the client and now re-connecting my personal onedrive doesn't work either.

I get the same error. Attached is debug output of attempting to connect back up to my personal onedrive account.
[2022-Mar11_not_successfully_authorised.txt](https://github.com/abraunegg/onedrive/files/8234279/2022-Mar11_not_successfully_authorised.txt)

Thank you for the help.

**Comment by abraunegg** [comment]
@0byt3 
Can you please 'clear' all your browser web cache and try the authentication process again.

**Comment by 0byt3** [comment]
Tried clearing all web cache, cookies, history and everything but I still have the same issue.
I normally use Firefox, so I downloaded and installed chromium and tried with that browser as well, but also have the same problem.

If it makes a difference: I am using Pop!_OS 21.10 (based on Ubuntu 21.10)

**Comment by abraunegg** [comment, marked as answer]
@0byt3 
I have just tested the client auth for Personal accounts locally:

```
[alex@centos7full onedrive]$ ./onedrive --confdir '~/.config/onedrive-personal/' --reauth
Configuration file successfully loaded
Deleting the saved authentication status ... re-authentication requested
Configuring Global Azure AD Endpoints
Authorize this app visiting:

https://login.microsoftonline.com/common/oauth2/v2.0/authorize?client_id=d50ca740-c83f-4d1b-b616-12c519384f0c&scope=Files.ReadWrite%20Files.ReadWrite.all%20Sites.Read.All%20Sites.ReadWrite.All%20offline_access&response_type=code&prompt=login&redirect_uri=https://login.microsoftonline.com/common/oauth2/nativeclient

Enter the response uri: https://login.microsoftonline.com/common/oauth2/nativeclient?code=M.R3_BAY.<redacted>

Application has been successfully authorised, however no additional command switches were provided.

Please use --help for further assistance in regards to running this application.

[alex@centos7full onedrive]$ 





```

So this is probably something 100% environmental to your setup / browser etc ..

**Comment by 0byt3** [comment]
I removed the existing files in ~/.config/onedrive (except for the config file) to start the from scratch, removed the --dry-run argument and added the --reauth argument. I was using the --dry-run argument to see if it would re-use the existing files that I have synchronized from the previous version. Client said it was authorised and and --synchronize is re-downloading everything, so it appears to be working now.

Thank you.
